### PR TITLE
Methods game update

### PR DESCRIPTION
**Pull request by iva-stoycheva**

### CODE REVIEW

File: src/dynamo.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ export class Dynamo {
         lastHeartbeat: new Date().getTime()
       })
     }
-    await client.putItem(params)
+    const result = await client.putItem(params)
+    console.log(result)
```
Comment (hsiliev) [inline src/dynamo.js:20]: Is `result` self-describing? Should we instead do something like `console.log("DynamoDB result", result")`?

Comment (iva-stoycheva) [reply src/dynamo.js:20]: Sorry. We tried something with Slavi and I forgot to delete it. Actually, we don't need console.log().

File: src/dynamo.js [hunk 2]
```diff
@@ -50,15 +50,16 @@ export class Dynamo {
...
+  async createGame(players) {
     const id = crypto.randomUUID()
     const params = {
       TableName: TABLE_GAMES,
       Item: marshall({
         id: id,
-        participants: participants,
-        playerTurn: participants[Math.round(Math.random())],
-        state: STARTED
+        players: players,
```
Comment (SlavenGrowy) [inline src/dynamo.js:59]: Currently we are passing a string array to the players, which does not conform to the protocol. Here is the code we came up with in the code review:
```js
  async createGame(players) {
    const shuffledPlayers = players.sort(() => Math.random() - 0.5)

    const [first, second] = shuffledPlayers

    const playerOne = this.getPlayerById(first)
    playerOne.piece = X
    const playerTwo = this.getPlayerById(second)
    playerTwo.piece = O

    const params = {
      TableName: TABLE_GAMES,
      Item: marshall({
        id: crypto.randomUUID(),
        players: [playerOne, playerTwo],
        playerTurn: first,
        state: STARTED,
        board: [EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY]
      })
    }
    await client.putItem(params)
  }
```

**Review by SlavenGrowy — APPROVED** [review]
LGTM 🔥

**Pull request merged**